Fix link to R installation

danielfrg opened the pull request:
I think that link is supposed to go to the installing R page.

slopp commented:
Great catch @danielfrg  - can you add the rendered HTML file _site/snapshot.html site? 

For this type of website (distill RMD), the CI doesn't actually render the site, it just publishes it. In other places (blogdown) you'll see different behavior, but here we need the html files included in the PR.

danielfrg commented:
Ok, I updated it with the rendered version.

There were some duplicated extra `<p>` tags from rmarkdown so I removed those manually, not 100% sure why did that happen.
